App/Containers/Comments.tsx

```diff
@@ -79,12 +79,9 @@ class Comments extends Component<Props> {
 
 const mapStateToProps = (state: RootState): StateProps  => {
   const { viewingPhoto } = state.photoViewing
-  if (!viewingPhoto) {
-    throw new Error('no viewing thread or photo')
-  }
```

Comment:
In both these cases, I was throwing an exception here because I thought it should never happen. Bad idea because that crashes the JS and unmounts the entire UI.

Been digging into why `viewingPhoto` would ever be `undefined`... I can't figure it out, but these fixes should at least make it so the UI doesn't unmount.

Reply:
Hmm, might this be a legacy issue? I think originally we didn't have comments at all, so maybe this was there to catch those cases?